server/athenian/api/models/web/user.py

```diff
@@ -15,6 +15,7 @@ class User(Model):
     def __init__(
         self,
         id: Optional[str] = None,
+        login: Optional[str] = None,
```

Comment:
This must be covered in the tests + handled in `auth.py`. We can revert back if it is too much side work.

Reply:
Ah indeed! Sorry, as I said [here](https://github.com/athenianco/athenian-api/pull/334#issuecomment-627276698) I actually reverted it back from the spec, but then forgot to revert the model itself. I'll do when cleaning up the history since it's not required now and would require extra work.

Reply:
:+1: